src/i18n/i18n.js

```diff
@@ -0,0 +1,29 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+// Languages
+import { en } from "./en/en.js"
```

Comment:
Husky should take care of this? Maybe remerge staging into this.

Reply:
I remerged and no dice, what was this supposed to fix?

Reply:
semicolon at the end of this line. Line 2 has one, line 5 doesn't.